### DISCUSSION TITLE
Spotify links no longer working.

### DISCUSSION THREAD

**Discussion by FordFriedel** [fent/node-ytdl-core#1149] [Q&A]
Using YTDL through a discord bot to play music in voice channels.

Using Node 8.1.0

Within the last two months, spotify links have ceased to work using YTDL, I have attempted to use multiple versions of the package with no luck.  Prior to posting an issue I wanted to make sure there was nothing else I was overlooking.  

Another thing that I noticed is that spotify links are not working with other bots as well, so I am wondering if there was a change with spotify's api that prevents YTDL from picking it up. 

Any help appreciated

**Comment by mtripg6666tdr** [comment]
Spotify is currently (and will not) supported in this lib, so the actual cause will not this lib.

**Reply by FordFriedel** [reply]
Thats unfortunate. Thanks for the response!
